apps/src/templates/lessonOverview/ResourceList.jsx

```diff
@@ -98,6 +106,17 @@ export default class ResourceList extends Component {
           {')'}
         </span>
       )}
+      {isGDocsUrl(resource.url) && (
+        <DropdownButton
+          text={i18n.makeACopy()}
```

Comment:
@bethanyaconnor if you had another idea for what to call this button, we can still change it. Technically, we could argue that downloading a PDF is "making a copy", but it's mostly the 3rd option to make a copy on Google Docs that this is referring to. I just couldn't come up with anything that made more sense here.

Reply:
"Make a copy" sounds good, especially as I don't have a better idea either